src/common/BaseAnnotator.ts

```diff
@@ -41,6 +41,10 @@ export type Options = {
 
 export const CSS_CONTAINER_CLASS = 'ba';
 export const CSS_LOADED_CLASS = 'ba-annotations-loaded';
+export const ANNOTATION_CLASSES: { [M in store.Mode]?: string } = {
```

Comment:
Moved this here since BaseAnnotator is a parent of DocumentAnnotator and both files are referencing these classes.